avocado/core/virt.py

```diff
@@ -16,16 +16,13 @@
 Module to provide classes for Virtual Machines.
 """
 
-import logging
 import time
 from xml.dom import minidom
 
 import libvirt
 
 from . import remoter
 
-LOG = logging.getLogger('avocado.test')
```

Comment:
(I mean look at it from the other side, if you were to add the LOG message, would you first send a patch to create LOG and then patch to use it?)